ui/README.md

```diff
@@ -88,25 +88,35 @@ Follow these steps to create your basic Azure Tree:
         }
     }
     ```
-1. Instantiate a new instance of `AzureTreeDataProvider` in your extension's `activate()` method, passing the `SubscriptionTreeItem` type and `loadMoreCommandId`. The `loadMoreCommandId` maps the 'Load More...' node to the command registered by your extension.
+1. Instantiate an `AzureAccountTreeItem` in your extension's `activate()` method. In most cases this will be the root of your tree, so you should pass in `undefined` as the parent and your `SubscriptionTreeItem` type to provide children.
```

Comment:
I find the SubscriptionTreeItem/SubscriptionTreeItem naming confusing.  I'd call it a constructor function (http://www.javascriptkit.com/javatutors/oopjs2.shtml).